grammar.js

```diff
@@ -163,6 +163,12 @@ module.exports = grammar({
     keyword_preserve: _ => make_keyword("preserve"),
     keyword_unsigned: _ => make_keyword("unsigned"),
     keyword_zerofill: _ => make_keyword("zerofill"),
+    keyword_on_conflict: _ => make_keyword("on conflict"),
+    keyword_do_nothing: _ => make_keyword("do nothing"),
+    keyword_do_update: _ => make_keyword("do update"),
```

Comment:
create a `keyword_do` and reuse it, `keyword_update` already exists as well. Like above, create a sequence out of these.

Reply:
Done